service/src/main/java/uk/nhs/adaptors/oneoneone/cda/report/mapper/HealthcareServiceMapper.java

```diff
@@ -60,9 +67,9 @@ private HealthcareService mapSingleHealthcareService(
         }
 
         if (intendedRecipient.isSetReceivedOrganization()) {
-            POCDMT000002UK01Organization receivedOrganization =
-                intendedRecipient.getReceivedOrganization();
-            Organization organization = organizationMapper.mapOrganization(receivedOrganization);
+            POCDMT000002UK01Organization receivedOrganization = intendedRecipient.getReceivedOrganization();
+            Organization organization = organizationMapper.mapOrganization(informationRecipient);
```

Comment:
Why only informationRecipient is passed here? The old method still should be used, just modified to accept additional argument and use it.